packages/cashier/src/components/cashier-onboarding/cashier-onboarding-side-note.jsx

```diff
@@ -36,47 +35,34 @@ const CashierOnboardingSideNote = ({ currency, is_crypto, openRealAccountSignup,
     };
 
     return (
-        <div
-            className={classNames({
-                'cashier-onboarding-detail': isMobile(),
-            })}
-        >
-            <div
-                className={classNames({
-                    'cashier-onboarding-detail__div': isMobile(),
-                })}
-            >
-                <Text className='cashier-onboarding-side-note__text' color='prominent' weight='bold' sixe='xs' as='p'>
-                    {is_crypto ? (
-                        <Localize
-                            i18n_default_text='This is your {{currency_code}} account.'
-                            values={{ currency_code }}
-                        />
-                    ) : (
-                        <Localize
-                            i18n_default_text='Your fiat account currency is set to {{currency_code}}.'
-                            values={{ currency_code }}
-                        />
-                    )}
-                </Text>
-                <Text className='cashier-onboarding-side-note__text' size='xxs' as='p' data-testid='side-note-text'>
-                    {getSideNoteDescription()}
-                </Text>
-                {is_crypto && (
-                    <div
-                        className='cashier-onboarding-side-note__link'
-                        onClick={() => {
-                            setDepositTarget(routes.cashier_deposit);
-                            openRealAccountSignup('add_crypto');
-                        }}
-                    >
-                        <Text size='xxs' color='red'>
-                            <Localize i18n_default_text='Manage your accounts ' />
-                        </Text>
-                        <Icon icon='IcChevronRight' color='red' />
-                    </div>
+        <div>
+            <Text className='cashier-onboarding-side-note__text' color='prominent' weight='bold' sixe='xs' as='p'>
+                {is_crypto ? (
+                    <Localize i18n_default_text='This is your {{currency_code}} account.' values={{ currency_code }} />
+                ) : (
+                    <Localize
+                        i18n_default_text='Your fiat account currency is set to {{currency_code}}.'
+                        values={{ currency_code }}
+                    />
                 )}
-            </div>
+            </Text>
+            <Text className='cashier-onboarding-side-note__text' size='xxs' as='p' data-testid='side-note-text'>
```

Comment:
please check this link on wiki JS regarding naming convention of data-testid:
https://wikijs.deriv.cloud/Frontend/General-information/test_guide#:~:text=Make%20sure%20you%20apply%20correct%20names%20while%20using%20data%2Dtestid%20attribute.%20It%20is%20required%20to%20use%20snake_case%20and%20dt%20prefix%3A